### PR TITLE
Ember styleguide

### PR DESCRIPTION
**Pull request by afomi**
sharing early

work remaining: 
[x] consolidate /kss/styleguide.html.erb to /styleguide.hbs
[x] replace static html in /styleguide.hbs with Ember components
[x] note opportunities for Ember refactorings, but don't dive deep. Note them for the team.

### CODE REVIEW

File: app/assets/javascripts/controllers/styleguide_controller.js.coffee [hunk 1]
```diff
@@ -0,0 +1,4 @@
+ETahi.StyleguideController = Ember.Controller.extend
+  error: "Bad File"
+
+  # upload: null # Set by Route
```
Comment (mikem) [inline app/assets/javascripts/controllers/styleguide_controller.js.coffee]: Is this comment necessary?

File: app/assets/javascripts/controllers/paper_manage_controller.js.coffee [hunk 2]
```diff
@@ -1,6 +1,7 @@
 ETahi.PaperManageController = Ember.ObjectController.extend
   positionSort: ["position:asc"]
-  sortedPhases: Ember.computed.sort('phases', 'positionSort')
+  # is model.phases the same as phases here?
+  sortedPhases: Ember.computed.sort('model.phases', 'positionSort')
```
Comment (neo-tahi) [inline app/assets/javascripts/controllers/paper_manage_controller.js.coffee]: is `model.phases` the same as `phases` here?

File: app/views/layouts/styleguide.html.erb [hunk 3]
```diff
@@ -0,0 +1,157 @@
...
+    ETahi.StyleguideView = Ember.View.extend({
+      didInsertElement : function(){
+        this._super();
+        Ember.run.scheduleOnce('afterRender', this, this.afterRenderEvent);
+      },
+      afterRenderEvent : function(){
+        initStyleguide()
+      }
+    });
+  </script>
```
Comment (mikem) [inline app/views/layouts/styleguide.html.erb]: Consider pulling this script into its own file, though it's not critical. We can do this later if the need arises.

File: app/views/layouts/styleguide.html.erb [hunk 4]
```diff
@@ -0,0 +1,157 @@
...
+    .styleguide-example .styleguide-element .styleguide-modifier-name {
+      display: none;
+    }
+    .row {
+      padding-bottom: 50px;
+    }
+    #tabs {
+      margin-bottom: 20px;
+    }
+  </style>
```
Comment (mikem) [inline app/views/layouts/styleguide.html.erb]: Consider pulling this style sheet into its own file, though it's not critical. We can do this later if the need arises.

**Comment by mikem** [comment]
Looks good to go. :+1:

**Pull request merged**